examples/object_detection_2d/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "object_detection_2d"
+version = "0.1.0"
+description = "Example Kolena integration for 2D object detection"
+authors = ["Kolena Engineering <eng@kolena.io>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12"
+kolena = ">=0.76.0,<1"
+s3fs = "^2023.5.0"
+fsspec = "^2023.5.0"
+scikit-learn = "^1.1.2"
```

Comment:
Should we install `kolena[metrics]` instead of adding `scikit-learn` manually?

Reply:
Now that we're installing `kolena[metrics]` can we remove this explicit sklearn dependency?